lib/LaTeXML/Post/XSLT.pm

```diff
@@ -76,7 +76,33 @@ sub process {
   if (my $icon = $params{ICON}) {
     # Hmm.... what type? could be various image types
     $params{ICON} = '"' . $self->copyResource($doc, $icon, undef) . '"'; }
+
+  # move to the site directory when applying the stylesheet,
+  # since it can create additional files with <exsl:document>
+  my $sitedir  = $doc->getSiteDirectory;
+  my $orig_cwd = pathname_cwd();
+  pathname_chdir($sitedir);
```

Comment:
A sudden realisation: in principle, this is a breaking change. If somebody out there uses `document()` to read some data, this change of folder will break the workflow almost irreparably.

@brucemiller what do you think of using `exsl:document()` in the stylesheets (for manifests or other files the user might want to generate)? If that feels like a good strategy for the EPUB manifest, I can work on another solution that doesn't break the previous behaviour (most likely something like a function `f:dest(relative-path)` that resolves to the site directory – I'd have to think about it).

Reply:
Correction: `document(uri)` resolves `uri` relative to the stylesheet location, whereas `document(uri, node)` resolves `uri` relative to the path of the document containing `node`.

Hence, this PR is *not* a breaking change. It is appropriate to switch directory so that `exsl:document()` writes to the destination directory rather than other random places, and it won't change the behaviour of `document()`.